privacy.html

```diff
@@ -0,0 +1,91 @@
+      contact us.</p>
+    <p>In certain circumstances, you have the following data protection rights:</p>
+    <ul>
+      <li>The right to access, update or to delete the information we have on you.</li>
+      <li>The right of rectification.</li>
+      <li>The right to object.</li>
+      <li>The right of restriction.</li>
+      <li>The right to data portability</li>
+      <li>The right to withdraw consent</li>
+    </ul>
+    <h2>Data Access</h2>
+    <p>The only parties that have access to the data include Josh Davis (Seattle, WA) as well
+      as Google LLC (1600 Amphitheater Parkway, Mountain View, CA 94043, USA)</p>
+    <h2>Google Analytics</h2>
+    <p>The services contained in this section enable the Owner to monitor and analyze web traffic and can be used to keep track of User behavior. </p>
+    <p>Google Analytics is a service that analyzes web traffic and is provided by Google LLC (“Google”). The data is collected and used to prepare
+      reports on user activity and shares it with Google's services. The data collected includes but not limited to: referrer, page visit data
+      such as time of visit, duration of visit, web browser and OS details. Data is anonymized.</p>
+    <p>View more information about Google Analytics here: <a href="https://analytics.google.com/analytics/web/">Google Analystics Info</a></p>
+    <p>More details:</p>
+    <ul>
+      <li><strong>Data process location:</strong> United States</li>
+      <li><strong>Personal Data processed:</strong> Tracker, usage data</li>
+    </ul>
+    <h2>Cookies and Web Beacons</h2>
+    <p>Like any other website, Two Factor Auth uses 'cookies'. These cookies are used to store information including
+      visitors' preferences, and the pages on the website that the visitor accessed or visited. The information is used
+      to
+      optimize the users' experience by customizing our web page content based on visitors' browser type and/or other
+      information.</p>
+    <p>For more general information on cookies, please read <a
+        href="https://www.cookieconsent.com/what-are-cookies/">"What Are Cookies"</a>.</p>
+    <h2>Privacy Policies</h2>
+    <p>You may consult this list to find the Privacy Policy for each of the advertising partners of Two Factor Auth.</p>
+    <p>Third-party ad servers or ad networks uses technologies like cookies, JavaScript, or Web Beacons that are used in
+      their respective advertisements and links that appear on Two Factor Auth, which are sent directly to users'
+      browser.
+      They automatically receive your IP address when this occurs. These technologies are used to measure the
+      effectiveness of their advertising campaigns and/or to personalize the advertising content that you see on
+      websites that you visit.</p>
+    <p>Note that Two Factor Auth has no access to or control over these cookies that are used by third-party
+      advertisers.
```

Comment:
```suggestion
```
The site doesn't serve ads.

Reply:
Oh thanks. Like I said, I used a template for Google services. I missed this section on ads.